contracts/destroyable/Destroyable.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity ^0.6.0;
+
+import "openzeppelin-solidity/contracts/ownership/Ownable.sol";
+
+contract Destroyable is Ownable {
+    bool private authorization;
```

Comment:
Please make the name clearer, authorization can mean anything

Reply:
we choose it together....